data/assets/scene/solarsystem/interstellar/c-2019_q4_borisov.asset

```diff
@@ -27,7 +27,7 @@ local C2019Q4BorisovTrail = {
     Color = { 0.9, 0.9, 0.0 },
     StartTime = "2015 JAN 01 00:00:00",
     EndTime = "2050 JAN 01 00:00:00",
-    SampleInterval = 60 * 60 * 24 -- = 86 400 seconds in 1 day
+    SampleInterval = 24 * 60 * 60 -- 1 sample per 1 day
```

Comment:
Do we have an `openspace.time.secondsPerDay()` function? If not then maybe that could be useful here, and since we have one for a year why should we not have one for a day too? [This](https://github.com/OpenSpace/OpenSpace/blob/55e196e351b443007cc15aad7203686e85b7ff0e/include/openspace/util/timeconversion.h#L39) C++ constant can be useful to implement that.

Reply:
We have a function now